composer.json

```diff
@@ -29,29 +29,29 @@
         "sonata-project/block-bundle": "^3.18",
         "sonata-project/core-bundle": "^3.17.0",
         "sonata-project/exporter": "^1.11.0 || ^2.0",
-        "symfony/asset": "^3.4.30 || ^4.2",
-        "symfony/config": "^3.4.30 || ^4.2",
-        "symfony/console": "^3.4.30 || ^4.2",
-        "symfony/dependency-injection": "^3.4.30 || ^4.2",
-        "symfony/doctrine-bridge": "^3.4.30 || ^4.2",
-        "symfony/event-dispatcher": "^3.4.30 || ^4.2",
-        "symfony/expression-language": "^3.4.30 || ^4.2",
-        "symfony/form": "^3.4.30 || ^4.0",
-        "symfony/framework-bundle": "^3.4.30 || ^4.2",
-        "symfony/http-foundation": "^3.4.35 || ^4.2.12",
-        "symfony/http-kernel": "^3.4.30 || ^4.2",
-        "symfony/options-resolver": "^3.4.30 || ^4.2",
-        "symfony/property-access": "^3.4.30 || ^4.2",
-        "symfony/routing": "^3.4.30 || ^4.2",
+        "symfony/asset": "^4.2",
+        "symfony/config": "^4.2",
+        "symfony/console": "^4.2",
+        "symfony/dependency-injection": "^4.2",
+        "symfony/doctrine-bridge": "^4.2",
+        "symfony/event-dispatcher": "^4.2",
```

Comment:
Symfony 4.2 is out EOL at the end of this month: https://symfony.com/releases

I think we should require at least 4.3? Otherwise the event-dispatcher deprecations will be a pain to fix :smile:

Reply:
Might be ok indeed :+1: